.env.ci

```
@@ -43,6 +43,8 @@ REDIS_PORT=$REDIS_PORT
 REDIS_HOST=$REDIS_HOST
 
 # IPFS
+IPFS_CLUSTER_ROOT=
+IPFS_CLUSTER_PASSWORD=
```

@JGiter On some of the other entries, we have an $env var replacement. do we need that here too?
```suggestion
IPFS_CLUSTER_ROOT=$IPFS_CLUSTER_ROOT
IPFS_CLUSTER_PASSWORD=$IPFS_CLUSTERPASSWORD
```

---

I thought that to test some environment locally one will set secret values directly in .env.* Do you think .env better to populate from other source?